runtime/test/src/lib.rs

```diff
@@ -669,6 +669,103 @@ impl_runtime_apis! {
         }
     }
 
+    #[cfg(feature = "runtime-benchmarks")]
+            impl frame_benchmarking::Benchmark<Block> for Runtime {
+
+                fn benchmark_metadata(extra: bool) -> (
+                    Vec<frame_benchmarking::BenchmarkList>,
+                    Vec<frame_support::traits::StorageInfo>,
+                ) {
+                    use frame_benchmarking::{list_benchmark, Benchmarking, BenchmarkList};
+                    use frame_support::traits::StorageInfoTrait;
+                    use frame_system_benchmarking::Pallet as SystemBench;
+                    use pallet_configuration::Pallet as PalletConfigurationBench;
+
+
+                    let mut list = Vec::<BenchmarkList>::new();
+
+                    list_benchmark!(list, extra, frame_system, SystemBench::<Runtime>);
+                    list_benchmark!(list, extra, pallet_configuration, PalletConfigurationBench::<Runtime>);
+
+                    let storage_info = AllPalletsWithSystem::storage_info();
+
+                    return (list, storage_info)
+                }
+
+                fn dispatch_benchmark(
+                    config: frame_benchmarking::BenchmarkConfig,
+                ) -> Result<Vec<frame_benchmarking::BenchmarkBatch>, sp_runtime::RuntimeString> {
+                    use frame_benchmarking::{
+                        add_benchmark, BenchmarkBatch, Benchmarking, TrackedStorageKey,
+                    };
+
+                    use frame_system_benchmarking::Pallet as SystemBench;
+                    impl frame_system_benchmarking::Config for Runtime {}
+                    use pallet_configuration::Pallet as PalletConfigurationBench;
+
+                    let whitelist: Vec<TrackedStorageKey> = vec![
+                        // Block Number
+                        hex_literal::hex!(  "26aa394eea5630e07c48ae0c9558cef7"
```

Comment:
We should remove any whitelisted key or at least make sure they match the key in our runtimes.